istio-mixin/mixin.libsonnet

```diff
@@ -1,9 +1,14 @@
 {
   grafanaDashboards: {
-    'istio_control_plane_dashboard.json': (import 'dashboards/istio_control_plane_dashboard.json'),
-    'istio_mesh_dashboard.json': (import 'dashboards/istio_mesh_dashboard.json'),
-    'istio_performance_dashboard.json': (import 'dashboards/istio_performance_dashboard.json'),
-    'istio_service_dashboard.json': (import 'dashboards/istio_service_dashboard.json'),
-    'istio_workload_dashboard.json': (import 'dashboards/istio_workload_dashboard.json'),
```

Comment:
Per previous feedback about grouping/filtering by services, destinations, etc..

Would it be appropriate to have request specific, or service specific, or destination specific dashboards?

It seems like these dashboards which are deleted may have provided some of that functionality?

Reply:
Yup, took the advice from the redundant tables, I created a seperate dashboard for services

Reply:
for the other dashboards, Most of the relevant metrics are included in the main dashboard itself

Reply:
Looks good, the new service dashboard provides a nice separation of concerns. :)